core/modules/file/file.admin.inc

```diff
@@ -197,7 +198,7 @@ function file_type_form($form, &$form_state, $type = NULL) {
   $form['mimetypes'] = array(
     '#type' => 'textarea',
     '#title' => t('Media types'),
-    '#description' => t("This is the list of media types (MIME types) which will be aassociated with this file type. When files with these media types are uploaded, Backdrop will identify the file type based on its media type. You should not use the same media types in more than one file type. Enter one media type per line. <br/><a href='@media_types_info'>More information about media types</a>.", array('@media_types_info' => 'https://en.wikipedia.org/wiki/Media_type')),
+    '#description' => t("The media (MIME) type(s) to be associated with this file type. When uploading a file, Backdrop automatically identifies the file type based on its media type. If multiple file types have the same media type, you will be prompted to manually select the file type. <a href='@media_types_info'>More information about media types</a>.<ul><li>Enter one media type per line</li><li>Asterisks can be used as wildcards (e.g. 'image/*')</li></ul>", array('@media_types_info' => 'https://en.wikipedia.org/wiki/Media_type')),
```

Comment:
This looks great @BWPanda, thanks 🙏🏼 

Just a couple of things:
1. Lets switch the outer double-quotes in `t()` into single quotes, and the single quotes in `href=` into double ones. I am told that parsing is faster with single quotes, since php does not try to find any variables that need to be escaped that way.
2. Lets make life easier for translators by avoiding HTML markup in `t()`:
   - we can use `theme_item_list()` for the bullet list.
   - we can move the "More information about media types" link out of the main description and concatenate it with an `l()` instead.